composer.json

```diff
@@ -1,10 +1,28 @@
 {
-    "require-dev": {
-        "dealerdirect/phpcodesniffer-composer-installer": "^0.4",
-        "wimg/php-compatibility": "dev-master",
-        "wp-coding-standards/wpcs": "dev-develop"
-    },
-    "scripts": {
-        "lint": "phpcs"
-    }
+  "name": "wordpress/gutenberg",
+  "type": "wordpress-plugin",
+  "license": "GPL-2.0",
+  "description": "Prototyping since 1440. Development hub for the editor focus in core.",
+  "homepage": "https://wordpress.github.io/gutenberg/",
+  "keywords": [
+    "gutenberg", "wordpress", "editor", "wp", "react", "javascript"
+  ],
+  "support": {
+    "issues": "https://github.com/WordPress/gutenberg/issues"
+  },
+  "require-dev": {
+    "dealerdirect/phpcodesniffer-composer-installer": "^0.4",
+    "squizlabs/php_codesniffer": "2.9.x",
+    "wimg/php-compatibility": "dev-master",
+    "wp-coding-standards/wpcs": "^0.11.0"
+  },
+  "require": {
+    "composer/installers": "~1.0"
+  },
+  "scripts": {
+    "lint": "phpcs",
+    "post-install-cmd": [
+      "phpcs --config-set installed_paths ../../wp-coding-standards/wpcs/"
```

Comment:
With the DealerDirect plugin, this script is no longer needed (and it doesn't register PHPCompatibility standard anyway).

Reply:
Removed in 72471347f38b2d40e768d687fa9431d1514b44ef